configure.ac

```diff
@@ -238,6 +238,39 @@ AX_VALGRIND_DFLT([sgcheck], [off])
 # Add a target to run testsuite with valgrind.
 AX_VALGRIND_CHECK()
 
+# Requirements for running test-crypt-kat-gen.py.
+ENABLE_CRYPT_KAT_GEN="no"
+PYTHON_BIN="python3 python"
+
+for ver in `seq 0 1 10`; do
+  PYTHON_BIN="python3.$ver $PYTHON_BIN"
+done
+
+AC_PATH_PROGS([PYTHON], [$PYTHON_BIN], [])
+
+if test x"$PYTHON" != "x"; then
+  AX_PROG_PYTHON_VERSION([3.6], [ENABLE_CRYPT_KAT_GEN="yes"],
+    [AC_MSG_WARN([Python >= 3.6 is required for optional Makefile targets.])
+  ])
+
```

Comment:
This works because you set up the order of candidates in `PYTHON_BIN` to go from most to least likely to be the one we want.  That probably deserves a comment, so future maintainers don't have to work out why it's not necessary to loop over the contents of `PYTHON_BIN` calling `AC_PATH_PROG` and then `AX_PROG_PYTHON_VERSION` for each.

Reply:
Fixed.

Reply:
Fixed by adding a comment above.